pkg/algorithms/edivisive/edivisive.py

```diff
@@ -16,15 +16,34 @@ class EDivisive(Algorithm):
     def _analyze(self):
         self.dataframe["timestamp"] = pd.to_datetime(self.dataframe["timestamp"])
         self.dataframe["timestamp"] = self.dataframe["timestamp"].astype(int) // 10**9
-        series= self.setup_series()
+        series = self.setup_series()
         change_points_by_metric = series.analyze().change_points
 
+
+        # Process if we have ack'ed regression
+        ackList = []
```

Comment:
hm. Not sure I like the idea of string parsing, because if we have a component that ever has a `_`

Reply:
Regardless of the characters that a component has in its name, the join for index_componentName will always be unique within this code block (.i.e for example 0_etch_avg_cpu, 0__etch_avg_memory etc). Its the same idea previously used in map being represented differently as a string by just using `_` as a delimiter so that we can avoid on redundant loops to optimize time complexity.

Reply:
I think I understand better! Thanks!